roles/smartgateway/tasks/main.yml

```diff
@@ -24,6 +24,8 @@
       to_load: "{{ True if 'events' in service_type else False }}"
     - name: ceilometer-metrics-configmap.yaml.j2
       to_load: "{{ True if 'metrics' in service_type and amqp_data_source == 'ceilometer' else False }}"
+    - name: sg-core-configmap.yaml.j2
```

Comment:
I feel like we're going to run in to a problem here at some point with this naming convention. Isn't Martin currently working on porting the Ceilometer metrics in to sg-core refactor as well? What about when we port over ceilometer and collectd events to sg-core?

Should we make this slightly less generic?

Reply:
Yes that will be a clash. In fact, events code has already been merged into the refactor branch and ceilometer metrics will be soon. This PR will need to be updated